src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -131,7 +152,7 @@ public void register(Router router) {
     final MoveItemIntoStatusService moveItemIntoStatusService = new MoveItemIntoStatusService(storage
       .getItemCollection(webContext), clients);
 
-    return moveItemIntoStatusService.processMarkItemInProcess(webContext)
+    return moveItemIntoStatusService.processMarkItem(webContext, ItemStatusName.IN_PROCESS)
```

Comment:
I think we should name this service consistently with the chosen domain language, `mark as` rather than `move`.

```suggestion
    return markItemStatusService.markItemAs(IN_PROCESS, webContext)
```

Reply:
That's legacy code it will be deleted at all.

Reply:
Then why has it been changed in this pull request if it will be deleted?

Does that mean the `MoveItemIntoStatusService` class is going to be deleted?